### PR TITLE
11 large card component

### PR DESCRIPTION
**Pull request by alaa-yasin**
- [X] Created LargeCard Component and styled it
- [X] Resized Icon Component
- [X] Added Montserrat font
- [X] Customized default variables of antd library
- [X] Edited the ticket svg icon
- [X] Added importing DevDependencies rule to ``` .eslintrc.js ``` file

### CODE REVIEW

File: src/components/LargeCard/index.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Card, Button, Typography } from 'antd';
+import data from './data';
```
Comment (MaiUbeid) [inline src/components/LargeCard/index.jsx]:
```suggestion

import data from './data';
```
leave line between import local files and libraries

File: src/assets/icons/ticket.svg [hunk 2]
```diff
@@ -5,7 +5,7 @@
     </clipPath>
   </defs>
   <g id="Artboard_1" data-name="Artboard – 1" clip-path="url(#clip-Artboard_1)">
-    <rect width="100" height="100" fill="#fff"/>
+    <!-- <rect width="100" height="100" fill="#fff"/> -->
```
Comment (MaiUbeid) [inline src/assets/icons/ticket.svg]:
```suggestion
    <!-- <rect width="100" height="100" fill="transparent"/> -->
```
change the fill to be transparent is better than make it as comment

File: src/components/LargeCard/index.jsx (hunk 1 above)
Comment (alaa-yasin) [reply src/components/LargeCard/index.jsx]: Done!

File: config-overrides.js [hunk 3]
```diff
@@ -4,12 +4,31 @@ module.exports = override(
...
+      '@font-family': 'Heebo',
+      '@font-size-base': '10px',
+      '@heading-color': 'fade(#000, 100%)',
+      '@border-radius-base': '6px',
+      '@btn-font-weight': '500',
+      '@btn-padding-horizontal-base': '25px',
+      '@btn-default-border': '#7A49A0',
+      '@btn-shadow': '0px 4px 8px #0000001F',
+      '@btn-primary-shadow': '0px 4px 8px #0000001F',
+      '@btn-height-base': '5vh',
```
Comment (MaiUbeid) [inline config-overrides.js]: button style is not suitable in the card
check it's style

File: src/assets/icons/ticket.svg (hunk 2 above)
Comment (alaa-yasin) [reply src/assets/icons/ticket.svg]: done!

File: src/components/LargeCard/index.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    <Card
+      className="large-card"
+      cover={<Image id={data[0].id} className="large-card__image" />}
+      hoverable
+      actions={[
+        <p className="large-card__price">
+          from{' '}
+          <span className="large-card__price--pound"> £{data[0].price}</span>
+        </p>,
+        <Button type="default">
```
Comment (MaiUbeid) [inline src/components/LargeCard/index.jsx]: Button has no margin between it and the card border

Comment (MaiUbeid) [reply src/components/LargeCard/index.jsx]: Capture

File: config-overrides.js [hunk 5]
```diff
@@ -4,12 +4,31 @@ module.exports = override(
...
+      '@btn-border-width': '2px',
+      '@btn-default-color': '#7A49A0',
+      '@border-width-base': '2px',
+      '@card-shadow': '0px 8px 16px #0000000F',
```
Comment (MaiUbeid) [inline config-overrides.js]: We need to discuss shadow to be suitable for all cards

File: src/components/Icon/style.css [hunk 6]
```diff
@@ -1,4 +1,4 @@
 .icon {
-  width: 15px;
-  height: 15px;
+  width: 14px;
```
Comment (MaiUbeid) [inline src/components/Icon/style.css]: This change has to be in another branch with another issue not here!

File: config-overrides.js (hunk 5 above)
Comment (alaa-yasin) [reply config-overrides.js]: sure

File: src/components/LargeCard/style.css [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+  line-height: 25px;
+  padding-left: 18px;
+  position: absolute;
+  top: 20px;
+  right: 0;
+  clip-path: polygon(0 0, 100% 0, 100% 100%, 0 100%, 10% 50%, 10% 50%);
+}
+
+.ant-card-actions {
+  border-radius: 0 0 12px 12px;
```
Comment (MaiUbeid) [inline src/components/LargeCard/style.css]: As xd file radius has to be 18 not 12

**Pull request merged**